p04-mui-quotes/src/Components/Quote/Quotes.js

```diff
@@ -1,41 +1,105 @@
 import { Grid, IconButton, Typography } from '@mui/material'
-import React from 'react'
+import {React, useState, useEffect, useRef} from 'react'
 import CurrentQuote from './CurrentQuote'
+import axios from 'axios';
 
-import { Favorite, SkipNext, FavoriteBorderOutlined } from '@mui/icons-material';
+import { Favorite, SkipNext, FavoriteBorderOutlined, Pause, Timer } from '@mui/icons-material';
 
 import { QuoteActions, QuoteContainer, QuoteListContainer } from './Quotes.styles'
 import QuotesFavsList from './QuotesFavsList';
+import configData from '../../Configs/config.json';
 
 const Quotes = () => {
+  const [randomQuoteTimer, setRandomQuoteTimer] = useState(0)
+  const [content, setContent] = useState('')
```

Comment:
There is no problem setting the quote props on separated ways, just to let you know that you can set an object with all those props. example: 

```
const [quote, setQuote] = useState({
  author,
  content,
  year,
  tags,
});
```

both ways are correct